extractor_api.go

```diff
@@ -26,27 +27,34 @@ func NewExtractorAPI(extractor *Extractor) http.Handler {
 			return
 		}
 
-		postBody := make([]byte, 100000)
-		r.Body.Read(postBody)
+		postBody, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			errlog.LogFromClientRequest(map[string]interface{}{
+				"error":  fmt.Sprintf("Error reading post body: %v", err),
+				"status": 500,
```

Comment:
We're using an integer literal here, but using the `http.StatusInternalServerError` constant on the line below.  Would be good to consistently use one or the other.